app/(dashboard)/dashboard/page.tsx

```
@@ -62,189 +161,278 @@ export default function DashboardPage() {
+  useEffect(() => {
+    if (skillNodes.length > 0) {
+      const positioned = calculateNodePositions(skillNodes, 1200, 800, 150, 120);
+      setPositionedNodes(positioned);
+    }
+  }, [skillNodes]);
+
+  const handleNodeClick = React.useCallback((node: any) => {
+    const nodeType = node.id.split('-')[0];
+    switch (nodeType) {
+      case 'space':
+        const spaceId = node.id.split('-')[1];
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)